prep-css-display/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Display Properties</title>
+  <style>
+ div {
+border: 4px solid blue;
+
+
+}
+button {
+  display: inline-block;
+  border: 4px solid red;
+  height: 40px;
+  width: 200px;
+}
+ul {
+  border: 4px solid purple;
+
+}
+li{
+  border: 4px solid magenta;
+  display: inline;
+}
+
+  </style>
+</head>
+
+<body>
+  <div> I am a div and by default I am block, which means I take up all the space I can horizontally.</div>
+  <div>
+    I am a div but I have <span style="border: 4px solid green; height: 100px; width 100px;">span elements inside me</span>.
+    Span elements are <span style="border: 4px solid green; height: 100px; width: 100px;">display inline by default</span>,
```

Comment:
Hi Dana, for this exercise we want to use CSS code in the `<style></style>` element to style our spans instead of putting the styles directly on the elements in an attribute.